cukinia/common_security_tests.d/kernel.conf

```diff
@@ -32,5 +32,37 @@ as "SEAPATH-00174 - Randomize kstack offset in on" cukinia_cmd \
 as "SEAPATH-00175 - Disable slab usercopy fallback" cukinia_cmd \
     grep -q "slab_common.usercopy_fallback=N" /proc/cmdline
 
+#ANSSI-BP-028-R8 l1tf test made in spectre_mitigations.conf
```

Comment:
Move this indication in spectre_mitigations.conf.
If spectre_mitigations.conf is modified, your comment will become wrong. If it is in spectr_mitigations.conf, the developper will easily see if it needs to be changed.